src/main/java/taxi/controller/page/CreateDriverController.java

```diff
@@ -0,0 +1,53 @@
+package taxi.controller.page;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.exception.AuthenticationException;
+import taxi.lib.Injector;
+import taxi.model.Driver;
+import taxi.service.AuthenticationService;
+import taxi.service.DriverService;
+
+public class CreateDriverController extends HttpServlet {
```

Comment:
move input data checks, salting and other logic to the service layer, controllers should only parse input and send it to the service, check other controllers